src/components/Tag/Tag.tsx

```diff
@@ -22,16 +22,12 @@ export const stylesByColor = {
 
 type Props = {
   /**
-   * The color of the tag. New colors should be supported in the flow type,
-   * in the stylesByColor object, and by its separate style in CSS file.
+   * The color variant of the tag. New variants should be supported in the flow type,
+   * in the VARIANTS array, and by its separate style in CSS file.
```

Comment:
nit: I know you're minimizing change here, but can we bring in the `VARIANTS` array now & remove "flow type" from the comment
```suggestion
   * The color variant of the tag. New variants should be supported
   * in the VARIANTS array and by its separate style in CSS file.
```

Reply:
Thanks! This was copied over from `next` so I'll update it there too.